consul/datadog_checks/consul/consul.py

```diff
@@ -372,6 +372,8 @@ def check(self, _):
                             tags.append('service:{}'.format(check['ServiceName']))
                     if check["ServiceID"]:
                         tags.append("consul_service_id:{}".format(check["ServiceID"]))
+                    if check["Node"]:
```

Comment:
can this cause a KeyError?

Reply:
The API output emits those keys as empty strings if the field is empty, and the `Node` option will show the Docker container ID if the node name is not defined:
```
{
        "Node": "07448c6bdb5f",
        "CheckID": "serfHealth",
        "Name": "Serf Health Status",
        "Status": "passing",
        "Notes": "",
        "Output": "Agent alive and reachable",
        "ServiceID": "",
        "ServiceName": "",
        "ServiceTags": [],
        "Type": "",
        "Definition": {},
        "CreateIndex": 20,
        "ModifyIndex": 20
    },
...
```